descriptions/css/properties/align-content.json

```diff
@@ -2,7 +2,7 @@
   "css": {
     "properties": {
       "align-content": {
-        "__short_description": "The <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> <strong><code>align-content</code></strong> property sets how the browser distributes space between and around content items along the cross-axis of a <a href='https://developer.mozilla.org/docs/Web/CSS/CSS_Flexible_Box_Layout'>flexbox</a> container, and the main-axis of a grid container."
+        "__short_description": "The <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> <strong><code>align-content</code></strong> property sets the distribution of space around content items of a <a href='https://developer.mozilla.org/docs/Web/CSS/CSS_Flexible_Box_Layout'>flexbox</a> or <a href='https://developer.mozilla.org/docs/Web/CSS/CSS_Grid_Layout'>grid</a> container. It applies to a flexbox's cross-axis or a grid's tracks."
```

Comment:
I might be wrong but I'm not sure "grid's tracks" is correct - tracks in grid go in both directions. Perhaps you want "block axis"? (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Box_Alignment_in_CSS_Grid_Layout#The_two_axes_of_a_grid_layout)

Also I think in "around and between", the " and between" part is important.

Do you like: "The CSS align-content property sets the distribution of space between and around content items along the cross-axis of a flexbox or the block axis of a grid."

Reply:
I misunderstood how it worked with grid. And I liked your suggestion, but it didn't quite hit the first-sentence-length requirement. I adjusted it a little and hopefully this new one works for you (also updated on the wiki)